integration/tests/basic/rpc/test_rpc_estimate_gas.py

```diff
@@ -91,6 +91,7 @@ def test_eth_estimate_gas_with_big_int(self, contract_name, process_gas, reserve
         assert deploy_trx_big_gas.get("status"), f"Transaction is incomplete: {deploy_trx_big_gas}"
         assert estimated_gas >= int(deploy_trx_big_gas["gasUsed"]), "Estimated Gas < Used Gas"
 
```

Comment:
I think we don't need this test file, because we will get errors here (this gas counts actual for devnet only, because mainnet has another version)

Reply:
removed